MSAL/src/native_auth/controllers/reset_password/MSALNativeAuthResetPasswordControlling.swift

```diff
@@ -26,15 +26,20 @@
 
 protocol MSALNativeAuthResetPasswordControlling: AnyObject {
 
-    func resetPassword(parameters: MSALNativeAuthResetPasswordStartRequestProviderParameters) async -> ResetPasswordStartResult
+    typealias ResetPasswordStartControllerResponse = MSALNativeAuthControllerTelemetryWrapper<ResetPasswordStartResult>
+    typealias ResetPasswordResendCodeControllerResponse = MSALNativeAuthControllerTelemetryWrapper<ResetPasswordResendCodeResult>
+    typealias ResetPasswordSubmitCodeControllerResponse = MSALNativeAuthControllerTelemetryWrapper<ResetPasswordVerifyCodeResult>
```

Comment:
I think `ResetPasswordVerifyCodeResult` should be renamed to `ResetPasswordSubmitCodeResult`. I know this is from a previous PR but it stands out as wrong in this PR, especially in mapping lines like this.

Reply:
Done